test/sanity.js

```diff
@@ -172,6 +173,12 @@ transformFile('test/fixtures/test-export-default-as.jsx', {
   presets: ['airbnb'],
   plugins: [
     inlineReactSvgPlugin,
+    [transformTs, { isTSX: true }],
+    () => ({
+      pre() {
+        console.warn = (msg) => { throw new Error(`Got console.warn: ${msg}`); };
```

Comment:
```suggestion
        console.warn = function warn(msg) { throw new Error(`Got console.warn: ${msg}`); };
```
this seems worth adding to every test?